execution_engine_testing/tests/src/test/system_contracts/auction/distribute.rs

```diff
@@ -243,46 +244,90 @@ fn should_distribute_delegation_rate_zero() {
         delegator_2_delegate_request,
     ];
 
-    let mut timestamp_millis =
-        DEFAULT_GENESIS_TIMESTAMP_MILLIS + DEFAULT_LOCKED_FUNDS_PERIOD_MILLIS;
-
-    let mut builder = InMemoryWasmTestBuilder::default();
+    let mut builder = InMemoryWasmTestBuilder::new_with_production_chainspec();
 
-    builder.run_genesis(&DEFAULT_RUN_GENESIS_REQUEST);
+    builder.run_genesis(&PRODUCTION_RUN_GENESIS_REQUEST);
 
     // initial token supply
     let initial_supply = builder.total_supply(None);
+    let total_payout = builder.base_round_reward(None);
     let expected_total_reward = *GENESIS_ROUND_SEIGNIORAGE_RATE * initial_supply;
     let expected_total_reward_integer = expected_total_reward.to_integer();
+    assert_eq!(total_payout, expected_total_reward_integer);
 
     for request in post_genesis_requests {
         builder.exec(request).commit().expect_success();
     }
 
-    for _ in 0..5 {
-        builder.run_auction(timestamp_millis, Vec::new());
-        timestamp_millis += TIMESTAMP_MILLIS_INCREMENT;
+    for _ in 0..=builder.get_auction_delay() {
```

Comment:
This should have changed the outcome of the test. The production `auction_delay` is 1 while the test used 5. Did it?

Reply:
It did indeed change the outcome, however, I refactored the test to account for the change.